docs/validator/staking-overview.md

```diff
@@ -8,6 +8,10 @@ sidebar_label: Orientation
 
 This section introduces you to the validation process and some additional resources.
 
+NEAR Protocol is a Proof-of-Stake (PoS) blockchain, and validators are nodes that manage any new transactions, adding them to new blocks. Every new block provides revenues for the validator in form of NEAR tokens, which are sourced from transaction fees and protocol rewards. In order to become a validator, a node must stake a certain amount of NEAR tokens, which are used as a collateral against "equivocation", which means the malicious creation of multiple rival blocks at the same time. The tokens at stake cannot be spent until the validation cycle, called epoch, is over.
+
+The protocol automatically elects validators issuing an auction for each epoch: nodes bidding with the highest stake become validators, while any insufficient stake will be returned right before the epoch is ready to begin, and become again available to be spent. 
```

Comment:
grammar: issuing --> by issuing
The statement about insufficient stake is almost correct, except that:
* It will generally be returned at the beginning of the next epoch.
* It might not be returned if it comes from some validator who was staking in the previous epoch. To avoid confusion, it might be helpful to clarify that this is a new proposal from someone who was not validating before.